<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkSingleElementWidget</class>
 <widget class="QWidget" name="LinkSingleElementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QVBoxLayout" name="header_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rechercher dans le folio :</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="folio_combo_box"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="content_layout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_this">
       <property name="text">
        <string>Voir cet élément</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_linked">
       <property name="text">
        <string>Voir l'élément lié</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
